package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "masters-way",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start:web": "pnpm --filter 'mw-webapp' run start"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@firebase/analytics": "^0.10.1",
+    "@radix-ui/react-accordion": "1.1.2",
+    "@radix-ui/react-avatar": "^1.0.4",
+    "@radix-ui/react-dialog": "1.0.5",
+    "@radix-ui/react-icons": "1.3.0",
+    "@radix-ui/react-progress": "^1.0.3",
+    "@radix-ui/react-select": "^2.0.0",
+    "@radix-ui/themes": "2.0.3",
+    "@tanstack/react-table": "8.12.0",
+    "@testing-library/jest-dom": "6.4.2",
+    "@testing-library/react": "14.2.1",
+    "@testing-library/user-event": "14.5.2",
```

Comment:
it does not look like global dependencies, be careful with them, I think it will be better to move specific dependencies back to the webapp